frontend/src/features/public-form/components/FormPaymentPage/stripe/StripePaymentElement.tsx

```diff
@@ -10,18 +10,32 @@ import InlineMessage from '~components/InlineMessage'
 
 import { useEnv } from '~features/env/queries'
 
+import { PublicFormWrapper } from '../../PublicFormWrapper'
 import {
   CreatePaymentIntentFailureBlock,
   PaymentStack,
   PaymentSuccessSvgr,
 } from '../components'
+import { PaymentHeader } from '../components/PaymentHeader'
 import { useGetPaymentInfo } from '../queries'
 
 import { GenericMessageBlock, StripePaymentBlock } from './components'
 import { useGetPaymentStatusFromStripe } from './queries'
 import { StripeReceiptContainer } from './StripeReceiptContainer'
 import { getPaymentViewStates, PaymentViewStates } from './utils'
 
+const PaymentFormWrapper = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <PublicFormWrapper>
+      <Box py="1rem" w="100%">
+        <Container w="57rem" maxW="100%" p={0}>
```

Comment:
I think maxWith should be `57rem` and w to be `100%` cause we'll usually phrase it as "take all the available width, but grow until 57rem"